pilot/pkg/model/validation.go

```diff
@@ -129,15 +129,15 @@ func (descriptor ConfigDescriptor) Validate() error {
 		}
 		descriptorTypes[v.Type] = true
 		if v.ClusterScoped {
-			if _, exists := clusterMessages[v.MessageName]; exists {
-				errs = multierror.Append(errs, fmt.Errorf("duplicate message type: %q", v.MessageName))
+			if _, exists := clusterMessages[v.Collection]; exists {
```

Comment:
I don't 100% agree with this change; I think we should either
- change ClusterScoped to MeshScoped, but that would be a breaking change to ProtoSchema, so rather not
- add MeshScoped field to ProtoSchema with similar behaviour so that a messageType can be defined as one of
  - only namespace-scoped
  - namespace-scoped & cluster-scoped
  - namespace-scoped & mesh-scoped

However I'm happy to keep things as they are for now, as this is really only ever used in validation.

Reply:
I don't get your comment, this is essentially no longer used

Reply:
The resources are either namespaces or clustered, there is no mesh scoped.  This behaviour has not changes with the PR